src/game_api/script/usertypes/screen_lua.cpp

```diff
@@ -44,8 +44,6 @@ void register_usertypes(sol::state& lua)
     lua["Screen"]["as_screen_title"] = &Screen::as<ScreenTitle>;
     lua["Screen"]["as_screen_menu"] = &Screen::as<ScreenMenu>;
     lua["Screen"]["as_screen_options"] = &Screen::as<ScreenOptions>;
```

Comment:
are those even documented somewhere?

Reply:
Nope, but I've also never had to cast screens anyway

Reply:
Think the idea was to use it in like `set_post_render_screen` or something